engine/src/main/java/io/zeebe/engine/processor/ReProcessingStateMachine.java

```diff
@@ -144,12 +144,12 @@ public ReProcessingStateMachine(ProcessingContext context) {
   ActorFuture<Void> startRecover(final long snapshotPosition) {
     recoveryFuture = new CompletableActorFuture<>();
 
-    LOG.info("Start scanning the log for error events.");
+    LOG.trace("Start scanning the log for error events.");
     lastSourceEventPosition = scanLog(snapshotPosition);
-    LOG.info("Finished scanning the log for error events.");
+    LOG.trace("Finished scanning the log for error events.");
 
     if (lastSourceEventPosition > snapshotPosition) {
-      LOG.info(
+      LOG.debug(
```

Comment:
I personally would keep the level at info for this one, as it indicates a state transition of the broker processing.